modules/ding_ddbasic/modules/ding_ddbasic_opening_hours/ding_ddbasic_opening_hours.theme.inc

```diff
@@ -0,0 +1,260 @@
+<?php
+/**
+ * @file
+ * Ding DDBasic opening hours themes.
+ */
+
+/**
+ * A single library.
+ */
+function theme_ding_ddbasic_opening_hours_week($variables) {
+  $classes = implode(' ', $variables['classes_array']);
+  return '<div class="' . $classes . '">' . render($variables['table']) . '</div>';
+}
+
+/**
+ * Preprocessor for ding_ddbasic_opening_hours_week.
+ */
+function template_preprocess_ding_ddbasic_opening_hours_week(&$variables) {
+  $timespan = array(
+    strtotime('last monday', strtotime('+1 day')),
+    strtotime('next sunday', strtotime('-1 day')),
+  );
+  $today = strtotime('today');
+  $structured = array();
+  $categories = array();
+
+  if (!empty($variables['timespan'])) {
+    $timespan = $variables['timespan'];
+  }
+
+  // Get and sort all the instances in the given timespan.
+  $instances = opening_hours_instance_load_multiple(
+    array($variables['node']->nid),
+    date('Y-m-d', $timespan[0]),
+    date('Y-m-d', $timespan[1])
+  );
+
+  // Add closed days.
+  for ($day = $timespan[0]; $day <= $timespan[1]; $day += 86400) {
```

Comment:
We've introduced a constant, and used that to get rid of the magical number instead.